### ISSUE TITLE
Allow yarn run <script> to access node_modules/.bin directly

### ISSUE THREAD

**Issue by Haroenv** [yarnpkg/rfcs#24]
Currently if you want to use something like electron, gulp, or a bunch of other packages that could be installed globally too, to use the local version you'd need to add an explicit part to `package.json`, like 

```json 
"scripts": {
  "gulp": "gulp",
  "electron": "electron"
}
```

I'd like it if these were added by default (but could be overriden in `package.json`). There are some relevant issues in `npm`: 

* https://github.com/npm/npm/issues/6053#issuecomment-134103443
* https://github.com/npm/npm/issues/10413
* https://github.com/sindresorhus/xo-init/issues/3

Basically it means that in `npm` it wouldn't be easily achievable, but maybe it is in `yarn`?

Happy to hear feedback

**Comment by joshmanders** [comment]
You can use `yarn run` for not only scripts but also things in `node_modules/.bin/`

So if you install `gulp` as a devDependency you can execute the binary by running `yarn run gulp`

**Comment by Haroenv** [comment]
That's exactly what I wanted (and I'm sorry that this opened an issue for something that already works)

Thanks for everything! 🎉
